### PR TITLE
Complete website design

### PR DESCRIPTION
**Pull request by tonnytech**
Added styles to the website using CSS to make improve on the user interface and use experience

### CODE REVIEW

File: src/Components/Menu.module.css [hunk 1]
```diff
@@ -52,47 +52,47 @@
...
+    letter-spacing: 1.9px;
+  }
+
+  .Logo {
+    width: 2.813rem;
+    height: 2.813rem;
+    margin: 0 0 0 41.5rem;
+    padding: 0.875rem;
+    border: solid 1px blue;
+  }
```
Comment (elisha2kyakpo1) [inline src/Components/Menu.module.css]: - [ ] Kindly add a logo, as per the requirement, all your designs should match the given design.

File: src/Components/Menu.module.css [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
 
 .navElements {
-    display: flex;
-    gap: 0.8rem;
-    list-style: none;
-    padding-left: 0;
+  display: flex;
+  gap: 0.8rem;
+  list-style: none;
+  padding-left: 0;
```
Comment (elisha2kyakpo1) [inline src/Components/Menu.module.css]: - [ ] I advise you to match the design as much as possible. The typography, color, and everything should match the given design. For example, the progress indicator does not match the design. Kindly see the image below.

logo

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @tonnytech, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @tonnytech  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: no linter errors 
:+1: design

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**